spec/spec_helper.rb

```diff
@@ -65,6 +65,6 @@
 
   config.ignore_request do |request|
     # Ignore capybara requests within feature tests
-    request.uri =~ /__identify__/
+    request.uri =~ /(__identify__|example\.com)/
```

Comment:
Why are we ignoring requests to example.com? It this related to the trouble you were having with Webmock @tatyree ?